libafl_bolts/Cargo.toml

```diff
@@ -117,6 +118,27 @@ llmp_debug = ["alloc", "std"]
 ## Reduces the initial map size for llmp
 llmp_small_maps = ["alloc"]
 
+#! ### Stable SIMD features
+
+## Use the best SIMD implementation by our [benchmark](https://github.com/wtdcode/libafl_simd_bench)
+stable_simd = ["covmap_wide128", "simplify_map_wide256"]
+
+## Enable naive coverage map features, usually no auto-vectorization
+covmap_naive = ["alloc"]
+## Enable coverage map implemented by wide::u8x16, usualy the fastest
+covmap_wide128 = ["alloc", "wide"]
+## Enable coverage map implemented by wide::u32x4, almost as fast as covmap_wide128
+covmap_wide256 = ["alloc", "wide"]
+## Enable coverage map implemented std::simd, slightly slower
+covmap_stdsimd = ["alloc", "wide"]
```

Comment:
I don't like unique features. Can't we just compile in all of them and have the user choose one on feedback creation?